habitat/plan.sh

```diff
@@ -97,6 +98,7 @@ do_build() {
     build_line "Installing gems from git repos properly ..."
     ruby ./post-bundle-install.rb
     build_line "Installing this project's gems ..."
+    fix_interpreter "${pkg_prefix}/vendor/bin/*" "$_chef_client_ruby" ruby
```

Comment:
@tpowell-progress see https://chefio.atlassian.net/browse/CHEF-660?focusedCommentId=82405 for an explanation for `fix_interpreter`

Reply:
@smandaka I think you should remove the `fix_interpreter` on line 111